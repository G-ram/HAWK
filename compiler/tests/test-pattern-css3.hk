BEGIN{}
[@p@]{
  print(this);
}
END{}
