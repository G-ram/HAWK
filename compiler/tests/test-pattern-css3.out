innerhtml=Pokemon Type: Wood
nextChild=
id=
innerhtml=Known as one of the most devilishly attractive Indian males, he has been known to ensnare women with his long lashes. Ways to repel this predator: Weak Against: Clean Laundry, Soap, Detergent Strong Against: Dirty clothes, garbage, rotten food
nextChild=
id=
innerhtml=Standing at 5'9, Srikar was ranked the #1 shooting guard in India. Known for a deadly jumpshot but absolutely no speed, stamina, or muscle, he was widely regarded as a shooting threat that would add greatly to Columbia's limited offensive skillset. However, after coming to Columbia to play on the basketball team, Srikar sadly found out that American basketball is drastically different from Indian basketball. He sadly couldn't even make the girls intramural team. After a year of dedication and training, he was allowed to play with the Columbia Under-12 basketball team.
nextChild=
id=
innerhtml=A prestigious Rabi Scholar, Srikar attempted to take 9 classes but was ultimately shut down by Dean Simon Bird. Often found in the Hartley Computer lab, Srikar is known for working furiously into the night and disappearing during the day.
nextChild=
id=
innerhtml=Known as the world's smartest chocolate labrador, Srikar was born in India, lived in India, and raised in India. Fell in love with a whale but as we know a dog and whale love story is bound to end in failure.
nextChild=
id=
